### PR TITLE
Bugsnag process

### PR DESCRIPTION
**Pull request by lukehansell**
I wanted to document our Bugsnag process since it's something that we each do so infrequently. This should act as a guide as to what is expected of you when you work on Bugsnag, but not a definitive checklist.

Thoughts, comments, questions and amendments are all welcome

Also, this is the process as I understand it. I may have got it wrong/ missed info. Correct me, it's okay.
- [x] +1 I like it
- [x] + 2 I like it also
- [ ] + 3 I'm just following the crowd

### CODE REVIEW

File: bugsnag-process.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+The stats show you:
+- occurrences (how many times has the happened)
+- users (how many individual people has it happened to)
+- last 2 weeks (instance graph of last two weeks)
+- stage (what environment is this happening in)
+- severity (how important do we think this is).
+
+You can use the search form on the left hand side to limit what bugs you see:
+
+![](htps://cldup.com/hmbTaL9r6A-2000x2000.png =x100)t
```
Comment (Joezo) [inline bugsnag-process.md]: t

**Comment by SkylerAvery** [comment]
We need to add in a section that on the end of your bugsnag slot, to clear all old bugs that have not been appearing for more than 2 weeks or so. That way we can prevent the huge build up that myself, @rahulpatel and @andrewcashmore had when we were on bugsnag :)

**Comment by lukehansell** [comment]
Is that a standard action now @LouisPryer? I wasn't even aware of it. Can anyone second this? If so I'll add it in.

**Comment by robhuzzey** [comment]
+1 excellent stuff, thanks

**Comment by stuartstobie** [comment]
certainly seems like reasonable practice to remove tickets if the snag hasn't been seen for a fortnight, I'd give that a +1

File: bugsnag-process.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+## tl;dr
+- check Bugsnag regularly
+- check metrics in the morning and during the day
+- send an email at the end of the day with info about the above
+
+---------------
+
+The Bugsnag duties rotate from developer to developer on a weekly basis.
+
+Bugsnag is an important part of our developer role here. It allows us time to catch and resolve bugs and issues which we would not otherwise see or prioritise. This creates a better development environment for us all as well as improving the customers experience on their site.
```
Comment (stuartstobie) [inline bugsnag-process.md]: experience on OUR site

File: bugsnag-process.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+## Bugsnag
+
+You'll require our Bugsnag credentials. Ask the last person on bugsnag for these.
+
+### How do I find bugs?
+Errors on our site are reported to [Bugsnag](https://bugsnag.com/holiday-extras/trip-app-js/errors) and are visible in the dashboard.
+
+The errors are split by project and you can switch between these using the menu in the top left corner.
+
+![](https://cldup.com/BjzztadbNV-1200x1200.png)
```
Comment (pmcnr-hx) [inline bugsnag-process.md]: I recommend attaching the screenshots to this PR and linking from there, this way all the content will be hosted in GitHub without external dependencies.

**Comment by lukehansell** [comment]
okay, so I've added the bit about removing bugsnags (cleaning up) and I've moved the images to a local /images/bugsnag-process/ directory...

c.c. @pmcnr-hx @LouisPryer @stuartstobie @Joezo

**Comment by pmcnr-hx** [comment]
:+1:

**Pull request merged**